src/util/hass-mixins.html

```diff
@@ -110,4 +108,48 @@
     }
   });
 
+/* @polymerMixin */
+window.hassMixins.MediaPlayerMixin = Polymer.dedupingMixin(superClass =>
+  class extends window.hassMixins.LocalizeMixin(superClass) {
+    static get properties() {
+      return {
+        playerObj: Object,
+        primaryText: {
+          type: String,
+          computed: 'computePrimaryText(playerObj)',
+        },
+        secondaryText: {
+          type: String,
+          computed: 'computeSecondaryText(playerObj)',
+        },
+      };
+    }
+
+    computePrimaryText(playerObj) {
+      return playerObj.stateObj.attributes.media_title ||
+             this.localize('state', playerObj.stateObj.state);
```

Comment:
Since this is the only place where it is used, we could have this logic live in the component and let the media player class just have a mediaTitle prop or something. If that is undefined, use the localize inside the component.

Reply:
An the other hand you could use it in custom state cards or panels

Reply:
Adam, what do you think of this? That way we can have 1 less mixin and keep all media player logic in the media player class.

Reply:
Yeah, definitely agree, just have had a busy few days. I should be able to get to all of the comments tonight.